### PR TITLE
Added Operational Usecases 

### PR DESCRIPTION
**Pull request by gagansk**
This PR has the following contents:

- Operational Usecases in the docs/goals folder 

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1201802387286625

### CODE REVIEW

**Comment by gagansk** [comment]
@joachimweyl Please review this and approve the PR.

File: goals/operationalUsecases.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+1. ## **Managing Users**
```
Comment (joachimweyl) [inline goals/operationalUsecases.md]: suggest getting rid of the numbering for headings. Also, headings are big and bold they don't also need to be bolded.

File: goals/operationalUsecases.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    **1.1 Terms Acceptance**
```
Comment (joachimweyl) [inline goals/operationalUsecases.md]: suggest using the standard numbered list format for markdown for all of your numbering
```
1.
   1.
   1.
1.
```
Markdown automatically numbers for you if you just use 1.

File: goals/operationalUsecases.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+New users should be informed of the acceptable uses of the NERC OpenShift environment before they are granted any access to the environment.  This applies equally to PIs, administrators, and ordinary users.  New users should also be informed that operations data for this environment will be archived and used for research.  (Because the cluster cannot currently meet GDPR requirements for removing users' data records, users expecting or requiring GDPR protections should be notified that they should not use NERC OpenShift at this point in time.)
+    **1.2 Request a new account**
+- 1.2.A : As a new user from an MGHPCC consortium member University, I should be able to create an account for myself in the NERC OpenShift environment by registering into the  MGHPCC Shared Services Account Portal, also known as RegApp, to request basic NERC access.
```
Comment (joachimweyl) [inline goals/operationalUsecases.md]: suggest using numbed list not bullets for all of your lists

File: goals/operationalUsecases.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+    **6.1 Add/Remove and track hardware**
+As an administrator of the cluster, I should be able to add/remove cluster nodes. I should also be able to view all the nodes and their status.
+    **6.2 Track faulty hardware**
+As an administrator of the cluster, I should be able to view and track the list of faulty nodes that need to be replaced.
+7. ## **Upgrade**
+    **7.1 Establish OpenShift cluster upgrade process**
+As an administrator of the cluster, I should be able to follow a set of documented instructions to help me in upgrading to newer versions of OpenShift. The document should also establish the process to mitigate any issues that might arise during the upgrade.
+8. ## **Monitoring**
+    **8.1 Generate and share operations alerts**
+    - As an administrator of the cluster, I should be able to generate, view, and share operation alerts in OpenShift using the monitoring tools available in the NERC cluster and store them for 30 days.
```
Comment (joachimweyl) [inline goals/operationalUsecases.md]: use numbered list

File: goals/operationalUsecases.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+6. ## **Hardware Management**
+    **6.1 Add/Remove and track hardware**
+As an administrator of the cluster, I should be able to add/remove cluster nodes. I should also be able to view all the nodes and their status.
+    **6.2 Track faulty hardware**
+As an administrator of the cluster, I should be able to view and track the list of faulty nodes that need to be replaced.
+7. ## **Upgrade**
+    **7.1 Establish OpenShift cluster upgrade process**
+As an administrator of the cluster, I should be able to follow a set of documented instructions to help me in upgrading to newer versions of OpenShift. The document should also establish the process to mitigate any issues that might arise during the upgrade.
+8. ## **Monitoring**
+    **8.1 Generate and share operations alerts**
```
Comment (joachimweyl) [inline goals/operationalUsecases.md]: if you want to do bolding do it after the number for the numbered list, although I think you can remove bolding for lists. I don't see bolding on the original document

File: goals/operationalUsecases.md [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+    - As an administrator of the cluster, I should be able to generate, view, and share operation alerts in OpenShift using the monitoring tools available in the NERC cluster and store them for 30 days.
+    - Critical alerts should be automatically shared by being published to a Slack channel identified by NERC.
+    - Security events and alerts should be logged and stored for 90 days.
+    - Events and alerts should be archived for researchers' use.
+    **8.2 Logging for OpenShift**
```
Comment (joachimweyl) [inline goals/operationalUsecases.md]: double-check this when you make it a numbered list because currently, this does not start a new line.

**Comment by joachimweyl** [comment]
@gagansk please review the changes I made.

**Comment by gagansk** [comment]
@joachimweyl LGTM!

**Review by joachimweyl — APPROVED** [review]
With the new updates, this looks good. Although I might suggest linking to the original somewhere in the PR comments.

**Comment by gagansk** [comment]
@joachimweyl Original document can be found [here](https://docs.google.com/document/d/1dHHpWvJD4b1mdXTDQhgWhiGeOLY7N7FmwvgI5VB3tP0/edit?usp=sharing).

**Pull request merged**